docs/pages/reference/signatures-inversion.rst

```diff
@@ -0,0 +1,11 @@
+.. _reference-signatures-inversion:
+
+Signature inversion
+###################
+
+.. currentmodule:: signatory
+
+See [...] for a description of the insertion algorithm.
```

Comment:
This `...` looks like it needs filling in?

Reply:
Yes this is a typo. I have not yet put the paper on arXiv, I will add the reference when I do. Would that be ok to you?